test/e2e/tests/errors.spec.js

```diff
@@ -5,7 +5,8 @@ const FixtureBuilder = require('../fixture-builder');
 describe('Sentry errors', function () {
   async function mockSentry(mockServer) {
     return await mockServer
-      .forPost('https://sentry.io/api/0000000/store/')
+      .forPost('https://sentry.io/api/0000000/envelope/')
```

Comment:
As discovered when manually testing, the error data is now sent as part of a POST request to the `/envelope` endpoint